### PR TITLE
Use environment variables for tokens in Go and Ruby examples. 

### PR DESCRIPTION
**Pull request by andrewsomething**
Let's recommend good practices and make these consistent with the cURL examples.

### CODE REVIEW

**Review by dbrian57 — APPROVED** [review]
lgtm! Thanks for cleaning this up.

**Review by scotchneat — APPROVED** [review]
This looks good. 
What do you think about changing `pat` to `token` in the go examples to be consistent across languages?

**Comment by andrewsomething** [comment]
@scotchneat  Good idea. Updated.

File: specification/resources/certificates/examples/go/create_certificates.yml [hunk 1]
```diff
@@ -2,26 +2,28 @@ lang: Go
...
   func main() {
-      pat := "mytoken"
+      token := os.Getenv("DIGITALOCEAN_TOKEN")
 
-      client := godo.NewFromToken(pat)
+      client := godo.NewFromToken(token)
       ctx := context.TODO()
 
-      key, err := ioutil.ReadFile("/path/to/privkey1.pem")
+      key, err := ioutil.ReadFile("/tokenh/to/privkey1.pem")
```
Comment (scotchneat) [inline specification/resources/certificates/examples/go/create_certificates.yml:16]: You could try matching `pat :=` and `(pat)` instead.

**Review by scotchneat — CHANGES_REQUESTED** [review]
Oops. Looks like some extra lines were matched.

File: specification/resources/certificates/examples/go/create_certificates.yml (hunk 1 above)
Comment (andrewsomething) [reply specification/resources/certificates/examples/go/create_certificates.yml:16]: :facepalm:

**Review by scotchneat — APPROVED** [review]
:shipit:

**Pull request merged**